app.js

```diff
@@ -10,18 +10,14 @@ const app = express();
 app.use(express.json());
 
 app.use(cors());
+app.use(express.static(path.join(__dirname, 'build')));
 
-app.get('/', async (req, res) => {
-  res.status(200).send('Hello!');
-});
+app.use('/api', routes);
```

Comment:
Thoughts on this? (this isn't real, just testing GH functionality )

```suggestion
app.use('/api/notrealsuggestionjusttesting', routes);
```